modules/workflow-condition-parser/src/main/antlr4/org/opencastproject/workflow/conditionparser/antlr/WorkflowCondition.g4

```diff
@@ -0,0 +1,20 @@
+grammar WorkflowCondition;
+
+booleanExpression : booleanTerm ( OR booleanExpression )? ;
+booleanTerm : booleanValue ( AND booleanTerm )? ;
+booleanValue : ( NOT )* ( '(' booleanExpression ')' | relation | BOOL ) ;
+relation : relationOperand COMPARISONOPERATOR relationOperand ;
+relationOperand : atom ( NUMERICALOPERATOR atom )* | atom ;
+atom : '(' relationOperand ')' | NUMBER | STRING | BOOL ;
+
+AND: 'AND' ;
+OR: 'OR' ;
+NOT: 'NOT' ;
+NUMBER: '-'? [0-9]+ ('.' [0-9]+)? ;
+COMPARISONOPERATOR: '<=' | '<' | '==' | '>' | '!=' | '>=' ;
+NUMERICALOPERATOR: '+' | '-' | '*' | '/' ;
+BOOL: 'true' | 'false' ;
+VAR: '${' ('A'..'Z' | 'a'..'z' | '0'..'9') '}' ;
```

Comment:
This seems to restrictive: At least the underscore characters needs to be supported, too, as several WOHs created variables containing underscores. Likely, the minus character is in use, too.

Reply:
Actually `VAR` seems not to be used in the grammar at all. Can you please remove it?

Reply:
Ah, yes, thanks for the reminder. I was going to evaluate the variables at the very end at first, so I had to keep the variables. Now that turned out different, so the placeholder parsing can be removed.